src/module.ml

```diff
@@ -46,21 +36,14 @@ module Name = struct
   let to_local_lib_name s =
     Lib_name.Local.of_string_exn s
 
-  let basename n ~(ml_kind : Ml_kind.t) ~(syntax : Syntax.t) =
-    let ext =
-      match syntax, ml_kind with
-      | Reason, Intf -> ".rei"
-      | Reason, Impl -> ".re"
-      | OCaml, Intf -> ".mli"
-      | OCaml, Impl -> ".ml"
-    in
-    String.lowercase n ^ ext
+  let basename n ~(ml_kind : Ml_kind.t) ~(syntax : Dialect.t) =
+    String.lowercase n ^ Dialect.extension syntax ml_kind
 end
 
 module File = struct
   type t =
     { path   : Path.t
-    ; syntax : Syntax.t
+    ; syntax : Dialect.t
```

Comment:
Same, here should rename the field `dialect`

Reply:
Done.